components/feature/push/src/main/java/mozilla/components/feature/push/Connection.kt

```diff
@@ -235,6 +234,8 @@ internal class RustPushConnection(
             )
 
             return DecryptedMessage(scope, data)
+        } else {
+            return null
```

Comment:
Interesting that the previous compiler didn't catch this! I wonder if it did some implicit return null.